modApiServer/src/org/aion/api/server/ApiAion.java

```diff
@@ -769,13 +769,12 @@ protected long getRecommendedNrgPrice() {
         }
     }
 
-    // leak the oracle instance. NrgOracle is threadsafe, so safe to do this, but bad design
-    protected NrgOracle getNrgOracle() {
-        return NRG_ORACLE;
+    protected long getDefaultTxNrg() {
+        return 90_000L;
     }
```

Comment:
What's the reason for deleting this method? It is be using for measuring reasonable price for the recently blocks.